app/macos/main.js

```diff
@@ -105,6 +112,36 @@ ipcMain.on("typeEmoji", (_event, arg) => {
   robot.typeString(arg);
 });
 
+// Return filtered and sorted emojis based on a search query
+ipcMain.handle("getEmojisForSearchString", (_event, arg) => {
```

Comment:
Since the LRU Map is defined in main.js instead of index.js, it made sense to do the filtering and sorting here and pass along that array to index.js.